blob/service/api/v1/v1.go

```diff
@@ -45,7 +46,7 @@ func (r *Router) Routes() []*rest.Route {
 		rest.Delete("/v1/users/:userId/blobs", r.DeleteAll),
 
 		rest.Post("/v1/users/:userId/device_logs", r.CreateDeviceLogs),
-		rest.Get("/v1/users/:userId/device_logs", r.ListDeviceLogs),
+		rest.Get("/v1/users/:userId/device_logs", api.RequireServer(r.ListDeviceLogs)),
```

Comment:
Device logs should be accessible by clinics in clinical trials, so that endpoint is not exclusive to backend services.